extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -670,15 +671,20 @@ public void handle(AsyncResult<String> event) {
                 if (event.failed()) {
                     Throwable effectiveCause = event.cause();
                     if (effectiveCause instanceof BindException) {
-                        if ((sslConfig == null) && (httpServerOptions != null)) {
-                            effectiveCause = new QuarkusBindException(httpServerOptions.getPort());
-                        } else if ((httpConfiguration.insecureRequests == InsecureRequests.DISABLED) && (sslConfig != null)) {
-                            effectiveCause = new QuarkusBindException(sslConfig.getPort());
-                        } else if ((sslConfig != null) && (httpConfiguration.insecureRequests == InsecureRequests.ENABLED)
-                                && (httpServerOptions != null)) {
-                            effectiveCause = new QuarkusBindException(
-                                    List.of(httpServerOptions.getPort(), sslConfig.getPort()));
-                        }
+                        effectiveCause = new QuarkusBindException(
+                            (BindException)effectiveCause,
+                            new ArrayList<>(){{
+                                if ((sslConfig == null) && (httpServerOptions != null)) {
+                                    this.add(httpServerOptions.getPort());
+                                } else if ((httpConfiguration.insecureRequests == InsecureRequests.DISABLED) && (sslConfig != null)) {
+                                    this.add(sslConfig.getPort());
+                                } else if ((sslConfig != null) && (httpConfiguration.insecureRequests == InsecureRequests.ENABLED)
+                                           && (httpServerOptions != null)) {
+                                    this.addAll(List.of(httpServerOptions.getPort(), sslConfig.getPort()));
+                                }
+                            }}
+                        );
+                        LOGGER.error(effectiveCause.getMessage());
```

Comment:
Are we sure we want this logged here? I thought the error was properly propagated to the user with the `completeExceptionally` below? /cc @cescoffier

Reply:
I threw it in as a completeness thing, if we are sure it is reported then it's okay